setup/instance/inputs.go

```diff
@@ -94,6 +94,7 @@ created for you. Existing VPCs must satisfy the following constraints:
     - Have access to the public internet (through a NAT instance or gateway)
     - Have at least 1 public and 1 private subnet
     - Each subnet must be tagged with ["Tier": "Private"] or ["Tier": "Public"]
+    - The subnets in each tier cannot have overlapping availability zones
```

Comment:
What if you said "No more than one tagged subnet per tier per availability zone"?

Reply:
that's better, but still not great. let me think a bit

Reply:
what about "Each subnet must reside in a unique availability zone"?